docs/infrastructure/getting-started.asciidoc

```diff
@@ -2,6 +2,8 @@
 [[xpack-metrics-getting-started]]
 == Getting started with infrastructure monitoring
 
+To get started with the <<infra-ui, Infrastructure app in Kibana>>, you need to start collecting metrics data for your infrastructure.
```

Comment:
This link confused me a bit. I would expect this to link me to Kibana (which isn't possible), but it just links to the next page in the docs. Is it needed? Can we just remove it but leave the text?